website/docs/guides/traefik.md

```diff
@@ -79,6 +79,7 @@ services:
       - DATABASE_PASS=${TM_DB_PASS}
       - DATABASE_NAME=${TM_DB_NAME}
       - DATABASE_HOST=database
+      - DATABASE_SSLMODE=disable
```

Comment:
Since this is the default value, I vote not to include it in the documentation at this location.

Reply:
Good practise is to include the default action commented out
`#      - DATABASE_SSLMODE=disable  # defaults to disable
`